SU-HSDO/suhumsci
# STN-682:  HOTFIX Carousel caption credit width change

# READY FOR REVIEW

## Summary
This work is to update the styling for the Captions/Credits view mode to fix an issue where the caption text appears behind the overlay for the text box in a Carousel on Colorful. 

When the caption is 160 characters long the width of the caption can cause overlap with the overlay of the carousel on colorful. This work shortens the width of the caption/credit back to 50% of the image for the Colorful carousel paragraph layout only.

<img width="1186" alt="Screen Shot 2021-02-10 at 10 47 17 AM" src="https://user-images.githubusercontent.com/9355698/107996474-7d2c9b00-6fae-11eb-8e76-7e018202ae6a.png">

JIRA Card: https://sparkbox.atlassian.net/browse/STN-682

_JIRA Card Note: The fix mentioned in the JIRA card for Traditional carousel, was a discussion of z-indexing, not the same as the Colorful issue and was ultimately decided to not be an issue in our standup conversations with Alex and Tori._

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. npm run test
2. To check this locally go to: http://sparkbox-sandbox.suhumsci.loc/qa/carousel-hero-image
3. Make sure you have added captions through the media caption/credit field found on the images in the carousel.
Such as: http://sparkbox-sandbox.suhumsci.loc/media/1016/edit?destination=/node/25631
![image](https://user-images.githubusercontent.com/9355698/107996082-85380b00-6fad-11eb-97fc-a0abea9e66b0.png)
4. After you have verified that the caption applies correctly to the colorful carousel, make sure that this styling update did not affect other applications of the caption/credit, especially within the other hero area image slider paragraph bundles, such as Hero Image.
5. The change should also not affect the Traditional Carousel caption/credit as it was not affected by the overlay covering the caption/credit like Colorful was. **(This will be QA'd with Tori to make sure that they want to keep this as is and not updated for a shorter length like the Colorful carousel.)**
6. You can also verify this work on the dev environment as this branch has been updated to be QA'd there.
https://sparkbox-sandbox-dev.stanford.edu/carousel-hero-image

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
